torchgeo/datasets/esri2020.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Esri 2020 Land Cover Dataset."""
+
+import abc
+import glob
+import os
+from typing import Any, Callable, Dict, Optional
+
+from rasterio.crs import CRS
+
+from .geo import RasterDataset
+from .utils import download_url, extract_archive
+
+
+class Esri2020(RasterDataset, abc.ABC):
+    """Esri 2020 Land Cover Dataset.
+
+    The 'Esri 2020 Land Cover dataset
+    <https://www.arcgis.com/home/item.html?id=fc92d38533d440078f17678ebc20e8e2>'
+    consists of a global single band land use/land cover map derived from ESA
+    Sentinel-2 imagery at 10m resolution with a total of 10 classes.
+    It was published in July 2021 and used the Universal Transverse Mercator (UTM)
+    projection. This dataset only contains labels, no raw satellite imagery.
+
+    The 10 classes are:
+    1 No Data
+    2 Water
+    3 Trees
+    4 Grass
+    5 Flooded Vegetation
+    6 Crops
+    7 Scrub/Shrub
+    8 Built Area
+    9 Bare Ground
+    10 Snow/Ice
+    11 Clouds
```

Comment:
```suggestion
    The 10 classes are:
    
    1. No Data
    2. Water
    3. Trees
    4. Grass
    5. Flooded Vegetation
    6. Crops
    7. Scrub/Shrub
    8. Built Area
    9. Bare Ground
    10. Snow/Ice
    11. Clouds
```

Reply:
This is 11 classes, not 10. We should either start counting from 0 or remove "No Data"